platform/sso/sso_okta.mdx

```diff
@@ -0,0 +1,54 @@
+---
+id: sso_okta
+title: Single Sign-On (SSO) With Okta
+sidebar_label: SSO with Okta
+description: Learn how to set up Single Sign-On (SSO) using OpenID Connect with Okta.
+---
+
+import { ScreenshotWrapper } from '../shared/_ScreenshotWrapper';
+
+This guide will help you set up Single Sign-On (SSO) for your organization using OpenID Connect with Okta.
+
+:::info
+This feature is only available with a Enterprise subscription on Tolgee cloud. [Upgrade your plan](/pricing) to use this feature.
+:::
+
+## Set Up Okta
+
+First, you need to set up an **OpenID Connect application** in Okta. To do this, you need to log in to your Okta account
+and navigate to **Admin** panel. Then find the **Applications** section and click on **Create App Integration**:
+<ScreenshotWrapper src="/img/docs/platform/sso/sso_okta_create_app.png" />
+
+Now select **OIDC - OpenID Connect** as the Sign-in method and **Web Application**  click **Next**:
+
+<ScreenshotWrapper src="/img/docs/platform/sso/sso_okta_app_settings.png" />
+
+Fill in the **Application settings**:
+- Grant type: **Authorization Code**
+- Login redirect URIs: **https://app.tolgee.io/login/open-id/auth-callback** (if you use Tolgee cloud) or your Tolgee instance URL ending with `/login/open-id/auth-callback`
+e.g. `https://tolgee-frontend-<name>/login/open-id/auth-callback`
+
+Click **Save**. Here is how the settings can look like:
+
+<ScreenshotWrapper src="/img/docs/platform/sso/sso_okta_general_settings.png" />
```

Comment:
With recent changes, users also have to enable the refresh token. Otherwise, login will fail.

Reply:
Yes, you're right. I need to update docs according to new changes

Reply:
Yes, although I'm just noting down whenever I see something. We can fix all the docs issues once the implementation is done :)